pkg/loop/internal/relayer/pluginprovider/chainreader/chain_reader_test.go

```diff
@@ -327,10 +392,6 @@ func (f *fakeChainReader) GetLatestValue(_ context.Context, contractName, method
 			}
 		}
 		return types.ErrNotFound
-	} else if method == DifferentMethodReturningUint64 {
```

Comment:
What was the purpose of removing this? Is it not used in the tests?

Reply:
answered in another comment